### ISSUE TITLE
No blockers are shown

### ISSUE THREAD

**Issue by svscorp** [nbrown02/FlowViz-Jira#33]
![image](https://github.com/nbrown02/FlowViz-Jira/assets/475464/f2c317a3-ccba-4289-b672-3bdb8a6d8e44)

After importing data, all but "Blockers" graphs are visible. None of blockers graphs displaying any data.

I am wondering, what is a prerequisite for this to be shown? Is it a status in JIRA (i.e. I have "On Hold" status for items that are blocked) or link type "blocked by / blocking" that I also use in combination with "On Hold" status. I have also items that have flags "impediment". But looks like neither of this makes the data to be shown.

Any ideas?

**Comment by nbrown02** [comment]
Hi there,

Blockers only works with labels, as documented in the [wiki](https://github.com/nbrown02/FlowViz-Jira/wiki/4.-Blockers). Unfortunately, flags are not kept in the same fields between jira instances, so what it is called for my jira will not be the same for yours, that's why they don't show.

Statuses to reflect being blocked are really an anti-pattern so they won't work either.

**Comment by svscorp** [comment]
Hi @nbrown02, on the anti-pattern I'd like to ask what source do you use? From the flow perspective it's good to distinguish 'doing' and 'waiting'. In some JIRA instances I set 'ready for status' and 'status' itself, and that allows a good understanding of the flow performance. However, there is never enough opps for learning, therefore, pls share your source where using extra status is an anti-pattern for the situation, that work is done, but next stage is impossible to take into "doing" due to an impediment.

Another question, whether linking with "blocked by" is an option for using in analytics?

**Comment by svscorp** [comment]
Also, as the item is marked "wontfix", closing this issue.

**Comment by nbrown02** [comment]
_Blocked by_ is available in the API, in this quick example:

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/72d0505d-618d-4ab3-b9d9-33cc6b9bded4)

ZZZSYN-36 is blocked by ZZZSYN-37

If you go into Power Query and go to the 'Expanded Data' step, you can edit this to add "issuelinks" as another field:

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/1aab7be7-8167-4991-9120-29caa66f2d60)

If you expand to new rows and expand the columns you'll eventually get to something like this:

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/c2e4d041-8455-4985-a345-9bfd5dadbca1)

I then did the following to find out which item was blocked:

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/ea6e415d-ef0d-4a98-a30a-8ed46a77273f)

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/91bf9ac0-005d-4e1d-a466-670f406e9a14)

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/1134475a-286e-49e8-ad8b-63f18d473412)

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/6f517c07-241c-4ae0-b2f7-2cdd229afd78)

You could then modify the BlockedLabel if statement to add this column in and therefore it would appear in the 'current blocked items' chart

I'll add this to the template at some point if you can wait...I'd need to do more research around doing this for the other charts
